src/components/CharacterLimitMainText/CharacterLimit.tsx

```diff
@@ -20,7 +22,7 @@ function CharacterLimit(props: ModuleProps): ReactNode {
   function renderWithIcon(): ReactNode {
     return (
       <div className={svgColor} data-testid="characterLimitWithIcon">
-        {props.svg}
+        {props.svg ? <Icon icon={props.svg} size={24} /> : null}
```

Comment:
Se você quer que apareça o icon apenas quando tiver props.svg, você pode fazer dessa forma
```suggestion
        {props.svg && <Icon icon={props.svg} size={24} />}
```